apps/challenges/models.py

```diff
@@ -228,6 +228,7 @@ class StarChallenge(TimeStampedModel):
     """
     user = models.ForeignKey(User)
     challenge = models.ForeignKey(Challenge)
+    is_starred = models.BooleanField(default=False)
```

Comment:
Entry in this model will be made only when a user stars a particular challenge. Why an extra boolean field here ?